docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -145,11 +145,12 @@ sudo apt install ziti-edge-tunnel
 </TabItem>
 </Tabs>
 
-2. Place an enrollment token JWT file or identity config JSON file in `/opt/openziti/etc/identities`.
+2. Place a copy of the enrollment token JWT file in the `/opt/openziti/etc/identities` directory.
+2. Set the filemode and owner so that members of group `ziti` will be able to read and delete the token file.
 
   ```bash
-  sudo -u ziti tee /opt/openziti/etc/identities/ziti-id.jwt >/dev/null
-  # paste the contents of the enrollment token JWT file and press Ctrl+D
+  sudo chown -cR :ziti        /opt/openziti/etc/identities
+  sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
```

Comment:
why doesn't the installer take care of this? feels like it should to me. Also is there any way to condense all these ubuntu's into a singular example? They all appear __nearly identical__ except for one word (the distro)? Seems redundant to me?

Reply:
The installer sets the filemode on the dir. This instructs the user to create a new file, the enrollment JWT, which also needs to have permissions set for user "ziti." The easiest way to do that is set correct permissions recursively for the identities dir. Otherwise it would be necessary to either know the JWT filename in advance, or provide a placeholder like token.jwt which requires the user to substitute their own filename, an unnecessary step.

Reply:
> any way to condense all these ubuntu's into a singular example

This would probably work for the Ubuntus.

```bash
(
set -euxo pipefail

curl -sSLf https://get.openziti.io/tun/package-repos.gpg \
  | sudo gpg --dearmor --output /usr/share/keyrings/openziti.gpg

source /etc/os-release

echo 'deb [signed-by=/usr/share/keyrings/openziti.gpg] https://packages.openziti.org/zitipax-openziti-deb-stable ${UBUNTU_CODENAME} main' \
  | sudo tee /etc/apt/sources.list.d/openziti.list >/dev/null

sudo apt update
sudo apt install ziti-edge-tunnel
)
```

Reply:
proposed this consolidation in https://github.com/openziti/ziti-doc/pull/614